src/vs/workbench/browser/parts/compositeBarActions.ts

```diff
@@ -211,6 +213,9 @@ export class ActivityActionViewItem extends BaseActionViewItem {
 		this.badge = dom.append(container, dom.$('.badge'));
 		this.badgeContent = dom.append(this.badge, dom.$('.badge-content'));
 
+		// Active border + background
+		dom.append(container, dom.$('.active-item'));
```

Comment:
@misolori why is this needed? Note that the composite switcher is being used both in activity bar and panel area.

Reply:
In order for us to set a border + bg color, it needs a dedicated dom element as the activity bar already uses the `:before` element and bg color for activity bar foreground color. I can see if I can hide this when not in use on the activity bar.

Reply:
Wrapped this in an if statement so it's only shown on the activity bar

Reply:
Wrapped this in an if statement so it's only shown on the activity bar